### ISSUE TITLE
Add human-readable values to notifications

### ISSUE THREAD

**Issue by tim-lucida** [OrienteerBAP/Orienteer#448]
1. Create Class_A and Class_B
2. Create inverse link attributes between classes
3. Create some documents of the classes
4. Go to link attribute and edit to rename it
5. Press [Save]

AR:The field 'Employee.jobTitle' is immutable and cannot be altered. Field value is: [#41:0, #41:1, #41:0]

ER: values of the vertices is also showed (set or default document names)

**Comment by PhantomYdn** [comment]
Am I right that expectation to have not IDs but links to documents with names?

**Comment by tim-lucida** [comment]
Yes, you're right: not IDs but clickable values of that attributes.
On click on that value user should be navigated to the Document
